packages/familie-form-elements/src/familie-react-select/FamilieReactSelect.tsx

```diff
@@ -98,32 +99,39 @@ export const FamilieReactSelect: React.FC<IProps> = ({
     propSelectStyles,
     ...props
 }) => {
-    const id = `react-select-${label}`;
 
-    const hentSelectProps = () => ({
-        styles: {
-            ...navSelectStyles(feil, erLesevisning),
-            ...propSelectStyles,
-        },
-        id: id,
-        isDisabled: erLesevisning,
-        isClearable: !erLesevisning,
-        value,
-        placeholder: 'Velg',
-        noOptionsMessage: () => 'Ingen valg',
-        ...props,
-    });
+    // @ts-ignore
+    const propsWithoutStyles:Omit<StateManagerProps, "styles"> = props;
```

Comment:
Usikker på hvorfor vi vil fjerne styles her 🤔

Reply:
Vil fjerne styles for å unngå å potensielt overskrive styles som vi legger inn manuelt (det er en lovlig verdi i props). 

Typescript blir hissig hvis det er lov å videresende styles samtidig som vi definererer det eksplisitt. 

tar gjerne imot forslag til hvordan dette kan løses litt mer "smoot" 😬